index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <title>Newsweek-Replica</title>
+</head>
+<body>
+  <header>
+    <div class = "container-fluid">
+      <div class = "navbar-top row align-items-center"> 
+          <div class = "col-lg-4 time d-none d-lg-block">
+            Wed, Jan 20, 2021 
+          </div>
+          <div class = "col-lg-4 col-6 text-lg-center">
+            <div class = "main-title-lg d-none d-lg-block">
+              Newsweek
+            </div>
+            <div class = "main-title-md d-block d-lg-none">
+              Newsweek
+            </div>
```

Comment:
- [ ] Code is duplicated, use Bootstrap classes with media query syntax to accomplish the same